### ISSUE TITLE
Generating of report looks like broken.

### ISSUE THREAD

**Issue by MininAS** [jz-jess/RobotEyes#90]
I don't know why it happenes only now but I get other issue related with generate report.

Console output:
```
Exception in thread Thread-33:
Traceback (most recent call last):
  File "C:\Program Files\Python36\lib\threading.py", line 916, in _bootstrap_inner
    self.run()
  File "C:\Program Files\Python36\lib\threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Program Files\Python36\lib\site-packages\RobotEyes\report_generator.py", line 68, in generate_report
    folder_name)
  File "C:\Program Files\Python36\lib\site-packages\RobotEyes\report_generator.py", line 227, in make_test_table
    for filename in os.listdir(baseline_folder + os.path.sep + folder_name):
FileNotFoundError: [WinError 3] The system cannot find the path specified: "visual_images_original\\Arguments: [ 'SeleniumLibrary' | tolerance=5 | template_id='1' ]"
```

A little invesigation:  report_generator.py
![image](https://user-images.githubusercontent.com/48047242/114050636-1ed8b600-9895-11eb-9cac-7671861dd12a.png)

**Comment by jz-jess** [comment]
@MininAS Is your test a template test (data driven) one?

**Comment by jz-jess** [comment]
@MininAS ok, I will take a look. Did you by any chance edit the open_eyes function in the __init__.py file?

**Comment by jz-jess** [comment]
Also could you provide the robot framework version you are on?

**Comment by MininAS** [comment]
No, didn't any editing.

Python 3.6.8
robotframework=3.1.2
robotframework-seleniumlibrary=4.3.0
robotframework-eyes=1.5.7
selenium=3.141.0

**Comment by jz-jess** [comment]
Okay Thanks. I think the first element with msg tag is the list of Arguments passed to open_eyes. I think RF logs the Arguments by default. It's most likely a bug. I will fix this as soon as I get some time today or tomorrow. I'm planning to add tests so these type of issues can be avoided. However it is something that needs time. 
If you would like to help out by adding some regular Robot tests within a tests folder. Please feel free to do so

**Comment by MininAS** [comment]
I'll try to find time for that :)

**Comment by jz-jess** [comment]
Hi @MininAS, I'm trying to reproduce this issue, however I'm unable to. Everything is working fine in my local machine using the same version of robotframework and selenium as yours. Here is what the `open eyes` keyword looks like in my `output.xml`
`<kw name="Open Browser" library="SeleniumLibrary">
<doc>Opens a new browser instance to the optional ``url``.</doc>
<arguments>
<arg>${url}</arg>
<arg>chrome</arg>
</arguments>
<msg timestamp="20210410 12:22:27.305" level="INFO">Opening browser 'chrome' to base url 'https://www.google.com/'.</msg>
<status status="PASS" starttime="20210410 12:22:27.304" endtime="20210410 12:22:31.802"></status>
</kw>
`
As you can see there is only one element with `msg` tag. Can you share your `output.xml` with me, at least the `open eyes` keyword alone so I can see the xml generated for you and make the necessary fix.

**Comment by MininAS** [comment]
```
<kw name="Open Eyes" library="RobotEyes">
<arguments>
<arg>SeleniumLibrary</arg>
<arg>tolerance=5</arg>
<arg>template_id=${uid}</arg>
</arguments>
<msg timestamp="20210410 10:05:35.435" level="TRACE">Arguments: [ 'SeleniumLibrary' | tolerance=5 | template_id='1' ]</msg>
<msg timestamp="20210410 10:05:35.447" level="INFO">Compare_Bar_Chart_Rendering_1</msg>
<msg timestamp="20210410 10:05:35.447" level="TRACE">Return: None</msg>
<status status="PASS" starttime="20210410 10:05:35.435" endtime="20210410 10:05:35.447"/>
</kw>
```

**Comment by jz-jess** [comment]
@MininAS Ah understood! That explains why there are multiple msg elements. Let me fix this. 
For the timebeing, if you remove `loglevel`, report generation should work fine

**Comment by jz-jess** [comment]
@MininAS Released a fix in 1.5.8. Please upgrade. Let me know if it works

**Comment by MininAS** [comment]
Works fine with and without `loglivel`!!! Thanks a lot for fast support
